modules/variance.js

```diff
@@ -9,7 +9,7 @@ export default function variance(collection, iteratee, context) {
   if (typeof iteratee == 'number' && collection != null && typeof collection[0] != 'object') { 
     iteratee = null;
   }
-  iteratee = cb(iteratee, context);
+  iteratee = iteratee(iteratee, context);
```

Comment:
This needs to be `_.iteratee` in order to work.

```suggestion
  iteratee = _.iteratee(iteratee, context);
```